apps/fileworker/src/lib/components/UploadInfoCard.svelte

```diff
@@ -0,0 +1,69 @@
+<script lang="ts">
+	import { page } from '$app/state'
+	import { Card, GradientButton, Input } from 'flowbite-svelte'
+
+	let {
+		file_id,
+		filename,
+		deletionToken,
+		...rest
+	}: {
+		file_id: string
+		filename: string
+		deletionToken?: string
+		[key: string]: unknown
+	} = $props()
+
+	const url = $derived.by(() => {
+		let url = new URL(page.url)
+		url.pathname = `/${file_id}/${filename}`
+		return url
+	})
+
+	const urlText = $derived.by(() => {
+		return url.toString()
```

Comment:
For simple expressions you can just do `let ... = $derived(url.toString())`. Same for the curl example.